app/src/main/java/com/google/ytdl/UploadsListFragment.java

```diff
@@ -46,23 +49,35 @@
  *         Left side fragment showing user's uploaded YouTube videos.
  */
 public class UploadsListFragment extends Fragment implements ConnectionCallbacks,
-        OnConnectionFailedListener {
+        OnConnectionFailedListener, GoogleApiClient.ConnectionCallbacks, GoogleApiClient.OnConnectionFailedListener {
 
     private static final String TAG = UploadsListFragment.class.getName();
     private Callbacks mCallbacks;
     private ImageWorker mImageFetcher;
-    private PlusClient mPlusClient;
+    private GoogleApiClient mGoogleApiClient;
     private GridView mGridView;
+    private static Context mContext;
 
     public UploadsListFragment() {
     }
 
+    @SuppressLint("ValidFragment")
+    public UploadsListFragment(Context context) {
+        mContext = context;
+    }
+
+
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        mPlusClient = new PlusClient.Builder(getActivity(), this, this)
-                .setScopes(Auth.SCOPES)
```

Comment:
I think you do not need to put in the youtube scope here. The youtube scope is only used in the MainActivity.java file under:

            credential = GoogleAccountCredential.usingOAuth2(
                    getApplicationContext(), Arrays.asList(Auth.SCOPES));